### PR TITLE
highlights for week ending 2022-06-24

### PR DESCRIPTION
**Pull request by gianfranco-l**
weekly highlights for week ending 2022-06-24

### Please remember to

- [ ] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/changes/highlights/2022-06-24.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+## General
+
+
+## User interfaces
+
+
+## Apps
+
```
Comment (weatherhog) [inline src/content/changes/highlights/2022-06-24.md:17]:
```suggestion
- Loki [v0.4.0](https://github.com/giantswarm/loki-app/releases/tag/v0.4.0) updates to Loki Upstream Chart [v0.48.5](https://github.com/grafana/helm-charts/releases/tag/loki-distributed-0.48.5) and updates Loki to [v.2.5.0](https://github.com/grafana/loki/releases/tag/v2.5.0) This release has some breaking changes - please see the [Changelog](https://github.com/giantswarm/loki-app/blob/v0.4.0/CHANGELOG.md#changes-required-in-your-valuesyaml-file) for instructions. 
- Promtail [v.0.3.2](https://github.com/giantswarm/promtail-app/releases/tag/v0.3.2) updates the upstream chart to [v6.0.0](https://github.com/grafana/helm-charts/releases/tag/promtail-6.0.0) and Promtail to [v.2.5.0](https://github.com/grafana/loki/releases/tag/v2.5.0). This update contains some breaking changes, please see the [Changelog](https://github.com/giantswarm/promtail-app/blob/master/CHANGELOG.md#changes-required-in-your-valuesyaml-file) for instructions.
- Cert-Manager [v2.14.0](https://github.com/giantswarm/cert-manager-app/releases/tag/v2.14.0) We fixed the broken relative URLs in the Readme. Updates to Upstream Image [v1.7.2](https://github.com/cert-manager/cert-manager/releases/tag/v1.7.2) which completely removes cert-manager API versions v1alpha2, v1alpha3, and v1beta1. If you need to upgrade your resources, [this document](https://cert-manager.io/docs/installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources) explains the process. 
```

File: src/content/changes/highlights/2022-06-24.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## General
+
+
+## User interfaces
+
+
+## Apps
+- Loki [v0.4.0](https://github.com/giantswarm/loki-app/releases/tag/v0.4.0) updates to Loki Upstream Chart [v0.48.5](https://github.com/grafana/helm-charts/releases/tag/loki-distributed-0.48.5) and updates Loki to [v.2.5.0](https://github.com/grafana/loki/releases/tag/v2.5.0) This release has some breaking changes - please see the [Changelog](https://github.com/giantswarm/loki-app/blob/v0.4.0/CHANGELOG.md#changes-required-in-your-valuesyaml-file) for instructions. 
+- Promtail [v.0.3.2](https://github.com/giantswarm/promtail-app/releases/tag/v0.3.2) updates the upstream chart to [v6.0.0](https://github.com/grafana/helm-charts/releases/tag/promtail-6.0.0) and Promtail to [v.2.5.0](https://github.com/grafana/loki/releases/tag/v2.5.0). This update contains some breaking changes, please see the [Changelog](https://github.com/giantswarm/promtail-app/blob/master/CHANGELOG.md#changes-required-in-your-valuesyaml-file) for instructions.
```
Comment (weatherhog) [inline src/content/changes/highlights/2022-06-24.md:18]:
```suggestion
- Loki [v0.4.1](https://github.com/giantswarm/loki-app/releases/tag/v0.4.1) sets loki config.auth_enabled to false by default
- Promtail [v.0.3.2](https://github.com/giantswarm/promtail-app/releases/tag/v0.3.2) updates the upstream chart to [v6.0.0](https://github.com/grafana/helm-charts/releases/tag/promtail-6.0.0) and Promtail to [v.2.5.0](https://github.com/grafana/loki/releases/tag/v2.5.0). This update contains some breaking changes, please see the [Changelog](https://github.com/giantswarm/promtail-app/blob/master/CHANGELOG.md#changes-required-in-your-valuesyaml-file) for instructions.
```

File: src/content/changes/highlights/2022-06-24.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+- Loki [v0.4.1](https://github.com/giantswarm/loki-app/releases/tag/v0.4.1) sets loki config.auth_enabled to false by default.
+- Promtail [v.0.3.2](https://github.com/giantswarm/promtail-app/releases/tag/v0.3.2) updates the upstream chart to [v6.0.0](https://github.com/grafana/helm-charts/releases/tag/promtail-6.0.0) and Promtail to [v.2.5.0](https://github.com/grafana/loki/releases/tag/v2.5.0). This update contains some breaking changes, please see the [Changelog](https://github.com/giantswarm/promtail-app/blob/master/CHANGELOG.md#changes-required-in-your-valuesyaml-file) for instructions.
+- Cert-Manager [v2.14.0](https://github.com/giantswarm/cert-manager-app/releases/tag/v2.14.0) We fixed the broken relative URLs in the Readme. Updates to Upstream Image [v1.7.2](https://github.com/cert-manager/cert-manager/releases/tag/v1.7.2) which completely removes cert-manager API versions v1alpha2, v1alpha3, and v1beta1. If you need to upgrade your resources, [this document](https://cert-manager.io/docs/installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources) explains the process.
```
Comment (weatherhog) [inline src/content/changes/highlights/2022-06-24.md:19]:
```suggestion
- Cert-Manager [v2.14.0](https://github.com/giantswarm/cert-manager-app/releases/tag/v2.14.0) We fixed the broken relative URLs in the Readme. Updates to Upstream Image [v1.7.2](https://github.com/cert-manager/cert-manager/releases/tag/v1.7.2) which completely removes cert-manager API versions v1alpha2, v1alpha3, and v1beta1. If you need to upgrade your resources, [this document](https://cert-manager.io/docs/installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources) explains the process. 
- Grafana-app [v1.0.0](https://github.com/giantswarm/grafana-app/releases/tag/v1.0.0) Upgrade upstream chart from version 6.24.1 to 6.31.0, and grafana from 8.4.2 to 9.0.1. This release includes a small set of breaking changes that you can check [here](https://grafana.com/docs/grafana/latest/release-notes/release-notes-9-0-0/#breaking-changes). This release also allows some existing values to be templetized (tpl function) and adds a bit more configurability options (e.g. network policies).
```

**Comment by gianfranco-l** [comment]
can someone please approve this to be merged? @MarcelMue @weatherhog @piontec

**Review by weatherhog — APPROVED** [review]

**Pull request merged**